lib/make-disk-image.nix

```diff
@@ -10,7 +10,7 @@ let
   vmTools = pkgs.vmTools.override {
     rootModules = [ "9p" "9pnet_virtio" "virtio_pci" "virtio_blk" ] ++ nixosConfig.config.disko.extraRootModules;
     kernel = pkgs.aggregateModules
-      (with nixosConfig.config.boot.kernelPackages; [ kernel ]
+      (with pkgs.linuxPackages; [ kernel ]
```

Comment:
Can we make this configurable instead but default to the configured kernel?
I see the usecase of people building smaller kernel, but this is the exception
rather than the norm and people doing that will know how to fix it.
There are advantages if filesystem are created with the same kernel that there are used with afterwards.

Reply:
I would prefer some kernel options as shown here: https://github.com/nix-community/disko/pull/698/files#diff-d08d64ddbf5f8a384c8213d871167c6f55002970d7312c302b4bd441fafb5e8dR32